styles/variables-gallium.less

```diff
@@ -104,12 +104,12 @@
 // Popup
 // ---------------------------------------
 @agate-popup-min-width: 510px;
-@agate-popup-max-width: 70%;
 @agate-popup-border-width: 3px;
 @agate-popup-border-radius: 20px;
 @agate-popup-border: 3px solid transparent;
 @agate-popup-padding: (@agate-component-spacing * 2.5) (@agate-component-spacing * 5);
-@agate-popup-title-padding: 0 (@agate-button-small-height) 0 0;
+@agate-popup-closebutton-title-padding-left: 0;
+@agate-popup-closebutton-title-padding-right: @agate-button-small-height;
```

Comment:
I think this type of change makes assumptions that each skin will have a close button that borrows space from the title. Is there a different approach we could use that leverages the existing variables?

Reply:
I don't know. All skins used this padding so far when a close button was present. The only thing I did was to break this padding into padding-left and padding-right to support RTL